### DISCUSSION TITLE
2021-07-09-webstorm

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#116] [General]
# 你居然还在用 WebStorm ？ - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Tool那么到底 WebStorm 更香在哪儿，周末花了个下午怼篇文章来开一个 “历史倒车轮“，给大伙介绍一下「如何用上 WebStorm 最新正版？」「如何让 WebStorm 好用不卡？」，以及「WebStorm 香的地方在哪里？」

https://tw93.fun/2021-07-09/webstorm.html

**Comment by DoubleLazyZ** [comment]
博主您好，想要尋問如果是處理 i18n 的問題，有推薦的 webstorm plugin 嗎？ 還是 i18n 處理會使用 vscode 比較好。

**Reply by tw93** [reply]
i18n 问题我理解会不会和编辑器关系不大，更多是扩展，或者说命令行的能力

**Reply by DoubleLazyZ** [reply]
了解，謝謝博主回覆，從這篇文章知道很多不知道的功能

**Comment by zvv001** [comment]
标题 二  3、4 的配图重复且与标题不符。
